### ISSUE TITLE
RPC: Broken `gasUsed` property in response

### ISSUE THREAD

**Issue by sammy007** [ethcore/parity#1111]
Testnet.
```bash
curl -X POST http://127.0.0.1:8545 -d '{"jsonrpc":"2.0","method":"eth_getTransactionReceipt","params":["0xe618759714ee896047613189370d1801403754900c91f537cec876beea2ca8bb"],"id":0}'
```

Geth: "gasUsed":"0x35e67"

Parity: "gasUsed":"0x056e73"

Well, I see that digits are the same actually, but their order is not :)

**Comment by sammy007** [comment]
Actually many properties are broken.

Geth: "blockNumber":"0xed6a9","cumulativeGasUsed":"0x56e73"
Parity: "blockNumber":"0x0ed6a9","cumulativeGasUsed"

Like in my previous issue https://github.com/ethcore/parity/issues/1090

**Comment by trapp** [comment]
I my case gasUsed has the same value as cumulativeGasUsed:

```
curl -X POST --data '{"jsonrpc":"2.0","method":"eth_getTransactionReceipt","params":["0x638443176f530e9a2976de28f075bb6f2a38aacc87d21a6ff79c554245da5d3e"],"id":74}' http://localhost:8545
```

parity: "gasUsed": "0x0802c8"
geth: "gasUsed": "0x5208" (correct)

This leads to wrong mining reward calculations because the fee for this transaction will be wrong (gasPrice * gasUsed) which makes this kind of critical for anyone trying to use parity to process financial data.

**Comment by gavofyork** [comment]
fix in #1118
